css/components/_btn.scss

```diff
@@ -0,0 +1,81 @@
+
+.btn {
+  border: 1px solid transparent;
+  border-radius: .25rem;
+  color: $pure-white;
+  cursor: pointer;
+  display: inline-block;
+  font-weight: 400;
+  padding: 1px;
+  position: relative;
+  text-align: center;
+  user-select: none;
+  vertical-align: middle;
+
+  a {
+    height: 100%;
+    width: 100%;
+  }
+
+  a,
+  a:hover,
+  a:active,
+  a:focus {
+    color: $pure-white;
+  }
+
+  >div {
```

Comment:
I find it really weird to have a `div` inside something having class of `btn`. What's the use case?